easybuild/easyconfigs/m/mpi4py/mpi4py-3.0.3-CrayGNU-19.10-python3.eb

```
@@ -0,0 +1,34 @@
+# Contributed by Theofilos Manitaras (cscs)
+# Victor Holanda (CSCS)
+easyblock = 'PythonPackage'
+
+name = 'mpi4py'
+version = "3.0.3"
+versionsuffix = '-python3'
+
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message
+Passing Interface (MPI) standard for the Python programming language, allowing
+any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'CrayGNU', 'version': '19.10'}
+
+source_urls = [BITBUCKET_DOWNLOADS]
+sources = [SOURCE_TAR_GZ]
+
+dependencies = [
+    ('PyExtensions', '3.6.5.7'),
+]
+
+# force rebuilding everything, including patched files
+buildopts = '--force'
+
+req_py_majver = '3'
+req_py_minver = '6'
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python3.6/site-packages/%(name)s'],
```

I think this is better to prevent possible mistakes in automatically updated easyconfigs for future upgrades of the system.
```suggestion
    'dirs': ['lib/python%s.%s/site-packages/mpi4py' % (req_py_majver, req_py_minver)],
```
If you could keep all three variables even better.

---

Used slightly different solution to keep all 3 variables.